app/src/components/common/question_input/modal_question.tsx

```diff
@@ -56,11 +56,19 @@ export const ModalQuestion = (props: Props) => {
 
   const { realitio } = useContracts(context)
 
-  const [questionId, setQuestionId] = useState<string>('')
+  const [questionURL, setQuestionURL] = useState<string>('')
   const [isSpinnerOn, setSpinnerOn] = useState<boolean>(false)
 
+  const extractId = (questionURL: string): string => {
+    const reQuestionId = /0x.*/
```

Comment:
I would like this regex to be a little safer. What about:

```
/question\/(0x[0-9A-Fa-f]{64})/
```

(We know the expected length because [`questionId` is a bytes32](https://github.com/realitio/realitio-contracts/blob/d7e986c9cba3445680430ea942e49920465f56f9/truffle/contracts/Realitio.sol#L252))

Then we extract the matching group `questionMatch[1]`.